packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -2609,7 +2617,9 @@ describe('network stubbing', function () {
     })
 
     it('intercepts cached responses as expected', {
-      browser: '!firefox', // TODO: why does firefox behave differently? transparently returns cached response
+      // TODO: why does firefox behave differently? transparently returns cached response
+      // TODO(webkit): fix+unskip. currently fails in WK because cache is disabled
```

Comment:
Is disabling the cache in WebKit something Playwright automation does, or something we do?

Reply:
PW automation.